solr/test-framework/src/java/org/apache/solr/cloud/AbstractBasicDistributedZkTestBase.java

```diff
@@ -1505,9 +1509,10 @@ private void testANewCollectionInOneInstance() throws Exception {
     SolrClient client3 = collectionClients.get(2);
     SolrClient client4 = collectionClients.get(3);
 
-    waitForRecoveriesToFinish(
-        oneInstanceCollection, getCommonCloudSolrClient().getZkStateReader(), false);
-    assertAllActive(oneInstanceCollection, getCommonCloudSolrClient().getZkStateReader());
+    getCommonCloudSolrClient();
+    waitForRecoveriesToFinish(oneInstanceCollection, ZkStateReader.from(cloudClient), false);
+    getCommonCloudSolrClient();
+    assertAllActive(oneInstanceCollection, ZkStateReader.from(cloudClient));
```

Comment:
Can you explain why `getCommonCloudSolrClient()` is called twice here and why the return value is not used?

Reply:
@haythemkh I don't know what you were doing here but I'll fix this in a minute and push a commit.  Thanks @janhoy for reviewing so thoroughly!